manim/mobject/polyhedra.py

```diff
@@ -0,0 +1,361 @@
+    """A tetrahedron, one of the five platonic solids. It has 4 faces, 6 edges, and 4 vertices.
+
+    Parameters
+    ----------
+    edge_length:
+        The length of an edge between any two vertices.
+
+    Examples
+    --------
+
+    .. manim:: TetrahedronScene
+        :save_last_frame:
+
+        class TetrahedronScene(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
+                obj = Tetrahedron()
+                self.add(obj)
+    """
+
+    def __init__(self, edge_length: float = 1, **kwargs):
+        unit = edge_length * np.sqrt(2) / 4
+        Polyhedron.__init__(
+            self,
+            vertex_coords=[
+                np.array([unit, unit, unit]),
+                np.array([unit, -unit, -unit]),
+                np.array([-unit, unit, -unit]),
+                np.array([-unit, -unit, unit]),
+            ],
+            faces_list=[[0, 1, 2], [3, 0, 2], [0, 1, 3], [3, 1, 2]],
+            **kwargs
+        )
+
+
+class Octahedron(Polyhedron):
+    """An octahedron, one of the five platonic solids. It has 8 faces, 12 edges and 6 vertices.
+
+    Parameters
+    ----------
+    edge_length:
+        The length of an edge between any two vertices.
+
+    Examples
+    --------
+
+    .. manim:: OctahedronScene
+        :save_last_frame:
+
+        class OctahedronScene(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
+                obj = Octahedron()
+                self.add(obj)
+    """
+
+    def __init__(self, edge_length: float = 1, **kwargs):
+        unit = edge_length * np.sqrt(2) / 2
+        Polyhedron.__init__(
+            self,
+            vertex_coords=[
+                np.array([unit, 0, 0]),
+                np.array([-unit, 0, 0]),
+                np.array([0, unit, 0]),
+                np.array([0, -unit, 0]),
+                np.array([0, 0, unit]),
+                np.array([0, 0, -unit]),
+            ],
+            faces_list=[
+                [2, 4, 1],
+                [0, 4, 2],
+                [4, 3, 0],
+                [1, 3, 4],
+                [3, 5, 0],
+                [1, 5, 3],
+                [2, 5, 1],
+                [0, 5, 2],
+            ],
+            **kwargs
+        )
+
+
+class Icosahedron(Polyhedron):
+    """An icosahedron, one of the five platonic solids. It has 20 faces, 30 edges and 12 vertices.
+
+    Parameters
+    ----------
+    edge_length:
+        The length of an edge between any two vertices.
+
+    Examples
+    --------
+
+    .. manim:: IcosahedronScene
+        :save_last_frame:
+
+        class IcosahedronScene(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
+                obj = Icosahedron()
+                self.add(obj)
+    """
+
+    def __init__(self, edge_length: float = 1, **kwargs):
```

Comment:
again, `edge_length` shouldn't it be `float` or `int`?

Reply:
changed